JetNews/build.gradle

```diff
@@ -37,7 +37,7 @@ plugins {
 subprojects {
     repositories {
         maven {
-            def snapshot = "7033025"
+            def snapshot = ""
             url  "https://androidx.dev/snapshots/builds/$snapshot/artifacts/repository/"
```

Comment:
This would be a 404, in `main` we don't have the maven entry so I'd just remove it completely.